arrow-libs/fx/arrow-fx-coroutines/src/main/kotlin/arrow/fx/coroutines/RacePair.kt

```diff
@@ -12,11 +12,14 @@ import kotlin.coroutines.intrinsics.suspendCoroutineUninterceptedOrReturn
 import kotlin.coroutines.intrinsics.intercepted
 import kotlin.coroutines.intrinsics.COROUTINE_SUSPENDED
 
+@Deprecated("Will be removed since it leaks Fiber, and breaks structured concurrency. Replace with select")
```

Comment:
what's `select` here?

Reply:
https://kotlinlang.org/docs/select-expression.html

Reply:
Ugh, I definitely prefer the `race` term :D Thank you!